### PR TITLE
fixing some syntactic changes

### PR DESCRIPTION
**Pull request by kishansagathiya**
## Tests

```sh
go test -tags integration github.com/ChainSafe/gossamer
```

## Primary Reviewer

@axaysagathiya @edwardmack

### CODE REVIEW

**Review by EclesioMeloJunior — APPROVED** [review]

File: dot/parachain/availability-store/availability_store.go
Comment (dimartiro) [inline dot/parachain/availability-store/availability_store.go:387]: I think is more clear to use the version directly 

```suggestion
		root:        trie.V0.MustHash(*tr),
```

And also I have a question, how do you know you should use the V0 to hash the trie?

**Review by dimartiro — CHANGES_REQUESTED** [review]

File: dot/parachain/availability-store/availability_store.go
Comment (kishansagathiya) [reply dot/parachain/availability-store/availability_store.go]: Looked more into this. It looks like the IDE thinks it is v1.
https://github.com/paritytech/polkadot-sdk/blob/7035034710ecb9c6a786284e5f771364c520598d/polkadot/erasure-coding/src/lib.rs#L278
Screenshot from 2024-02-12 12-39-14

LayoutV1 is for V1 trie layout.

Comment (dimartiro) [reply dot/parachain/availability-store/availability_store.go]: Actually using V0 is the same that using `MustHash` with `NoMaxInlineValueSize` https://github.com/ChainSafe/gossamer/blob/f5b9c4c89557d6332396de9e9fa3a8cf782f7259/lib/trie/layout.go#L94-L96

Or we can wait until merge this [PR](https://github.com/ChainSafe/gossamer/pull/3739) from Eclesio and we can get rid of this parameter

**Review by dimartiro — APPROVED** [review]

**Pull request merged**
